tornado_botocore/base.py

```diff
@@ -183,3 +206,4 @@ def call(self, callback=None, **kwargs):
             api_params=kwargs,
             callback=callback
         )
```

Comment:
if you are passing the callback into the method you should not yield it as it shouldn't be returning a future